local-helm/start-dashboard-proxy.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+echo "Starting dashboard proxy"
+
+TOKEN=`kubectl -n kube-system describe secret default | grep token: | awk '{print $2}'`
+kubectl config set-credentials docker-desktop --token=$TOKEN
+
+DASHBOARDURL="http://localhost:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/#!/overview?namespace=buyingcatalogue"
+echo "Copy"
```

Comment:
don't think you need this copy as the link echoed below should be clickable

Reply:
I think that depends on your terminal as mine wasn't clickable, but I can remove it if you like?

Reply:
mac 😄

Reply:
nah it is fine I just thought that was was bash did

Reply:
👍